blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md

```diff
@@ -0,0 +1,300 @@
+
+In addition, in the second half of July, the educational platform Pravaler became the first company in the sector to issue tokenized debentures in the local market, raising around U$ 12 million in the operation.
+
+In the Brazil market, there are already companies that have been operating in the area of tokenization of real assets for some years, converting them into NFTs, which stands for Non-Fungible Token, such as MB Tokens, a subsidiary of Mercado Bitcoin focused on the segment, and Hurst Capital.
+
+These companies tokenize and offer investors in small fractions anything that can be imagined, as long as it has some real asset behind it that serves as collateral - from consortium quotas to energy marketing contracts, passing through works of art, music royalties, and even the sale of soccer stars like Neymar and Gabriel Barbosa.
+
+These contracts of various kinds are registered on the blockchain network and fractionated, and then offered at affordable prices on cryptocurrency trading platforms to investors.
+
+The **[Celo hackathon to tokenize government assets in Brazil](https://patrimoniodauniaonaweb3.hackerearth.com/)** was an exciting initiative that aims to leverage the power of blockchain technology to promote transparency and efficiency in the management of public assets.
+
+The tokenization of government assets has the potential to transform the way public assets are managed around the world, by enabling greater transparency and accountability in the ownership, transfer, and management of these assets. By creating digital tokens that represent these assets, it becomes possible to track their ownership and transaction history in a secure and transparent way.
+
+To get started with **tokenizing physical assets on Celo**, you'll need to do some research and planning to determine what kind of asset you want to tokenize, how you want to structure your tokens, and what kind of smart contracts and other infrastructure you'll need to create. Once you have a plan in place, you can use tools like OpenZeppelin to help you execute your vision and bring your tokenized asset to life on the blockchain.
+
+## Mechanics of on-chain asset tokenization:
+
+**Build the ecosystem:** Service providers ensure that technology integration is seamlessly done with a large ecosystem of vendors so that issuers can create an end-to-end customized solution (e.g., custody, KYC/AML provider, legal, marketing, advisory, tax, etc.).
+
+**Register the underlying asset and set up the token:** The asset is registered and made available for tokenization on the blockchain network. Once the raise is structured and the ecosystem is mapped, service providers enable issuers to configure their tokens in a simple and intuitive way, regardless of the type of asset. The number of assets to be registered on the platform is a decision made in alignment between the issuer and the platform provider.
```

Comment:
**Register the underlying asset and set up the token:** The asset is registered and made available for tokenization on the blockchain network. Once the raise is structured and the ecosystem is mapped, service providers enable issuers to configure their tokens simply and intuitively, regardless of the type of asset. The number of assets to be registered on the platform is a decision made in alignment between the issuer and the platform provider.

Reply:
Done